apps/api-server/src/routes/api/status.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express');
+const db = require('../../db');
+const auth = require('../../middleware/sequelize-authorization-middleware');
+const pagination = require('../../middleware/pagination');
+
+let router = express.Router({ mergeParams: true });
+
+router.all('*', function (req, res, next) {
+  req.scope = [];
+  req.scope.push('defaultScope');
+  req.scope.push({ method: ['forProjectId', req.project.id] });
+
+  if (req.query.includeProject) {
+    req.scope.push('includeProject');
+  }
+
+  if (req.query.type) {
+    let type = req.query.type;
```

Comment:
Wat is de type van een status? Is dat niet alleen relevant bij tags of zijn er ook groepen voor statussen?

Reply:
Klopt, dat is meegekopieerd uit tags en hoort hier niet meer. Verwijderd.